lib/pkg/flow-base/lib/FlowProcess.js

```diff
@@ -342,7 +342,7 @@ export class FlowProcess {
       ...args,
       '--from', 'nuclide',
     ];
-    const pathToFlow = getPathToFlow();
+    const pathToFlow = await getPathToFlow();
```

Comment:
the difference is a `fileExists` check for which I don't expect to be a slow - however, I can instead opt to cache the values, I think it's pretty easy to get a callback from VS Code when your settings changes and so the stored path can be reset then - will take a look